docs/lib/storage/fragments/native_common/configureaccess/common.md

```diff
@@ -0,0 +1,46 @@
+ When adding the Storage category, you configure the level of access authenticated and guest users have to your S3 bucket. Additionally, when uploading files using the Storage category, you can specify the access level for that file to be either guest(public), protected, or private.
```

Comment:
Similar to my comment below: does "public" confuse users who are expecting any random person on the internet to be able to view an object uploaded with that access level?

```suggestion
 When adding the Storage category, you configure the level of access authenticated and guest users have to your S3 bucket. Additionally, when uploading files using the Storage category, you can specify the access level for that file to be either guest(public), protected, or private.
```

Reply:
updated the section below at line 7 to

> For guest access, the user of your application is not signed in, however you will still need [Authentication](~/lib/auth/getting-started.md) configured in your application to allow the user to assume an unauthenticated role from Cognito Identity Pool.
> 
> For protected and private access, the `[USER_ID]` below corresponds to the unique ID of the user. Once the user has signed in, the `[USER_ID]` can be retrieved from the session by accessing the identity id. See [Accessing credentials](~/lib/auth/access_credentials.md) to retrieve the identity id, and use this as the unique ID of the authenticated user.
>

Reply:
Thanks for that addition, it helps clarify. What about also removing the "(public)" altogether to further clarify?

Reply:
that sounds good